app/controllers/chargeback_controller.rb

```diff
@@ -294,16 +434,21 @@ def cb_rate_show
   # Delete all selected or single displayed action(s)
   def cb_rates_delete
     assert_privileges("chargeback_rates_delete")
-    rates = []
+    # we show the compute rates
+    rates_compute = []
     if !params[:id] # showing a list
-      rates = find_checked_items
-      if rates.empty?
+      rates_compute = find_checked_items
+      if rates_compute.empty?
         add_flash(_("No %s were selected for deletion") % ui_lookup(:models => "ChargebackRate"), :error)
         render :update do |page|
           page.replace("flash_msg_div", :partial => "layouts/flash_msg")
         end
       end
-      process_cb_rates(rates, "destroy")  unless rates.empty?
+      process_cb_rates(rates_compute, "destroy") unless rates_compute.empty?
+      # delete the storage rate (id_compute_rate + 1)
+      rates_storage = rates_compute.map { |v|; v + 1 }
```

Comment:
I don't understand this comment, neither my local rubocop nor the manageiq rubocop have problems with this line.